lib/matplotlib/backends/backend_wx.py

```diff
@@ -1128,14 +1128,21 @@ def _init_toolbar(self):
         self._parent = self.canvas.GetParent()
 
         self.wx_ids = {}
+        self.bitmaps = {}
```

Comment:
```suggestion
        self._bitmaps = {}
```
Please mark this as private.

Reply:
I've made this change in the latest commit.